prawns/staking/src/processor.ts

```diff
@@ -0,0 +1,53 @@
+import { SubstrateProcessor } from '@subsquid/substrate-processor';
+import stakingActionEvent from './handlers/staking.action.event';
+import stakingNominateCall from './handlers/staking.nominate.extrinsic';
+import { SubstrateNetwork, SubstrateStakingActionType } from './model';
+import { KnownArchives, lookupArchive } from '@subsquid/archive-registry';
+import { TypeormDatabase } from '@subsquid/typeorm-store';
+
+const supportedNetworks = ['kusama', 'polkadot'];
+const network: SubstrateNetwork = process.env.NETWORK as SubstrateNetwork;
+
+if (!supportedNetworks.includes(network)) {
+  throw Error('Network not supported');
+}
+
+new SubstrateProcessor(new TypeormDatabase())
+  .setBatchSize(500)
+  .setDataSource({
+    archive: lookupArchive(network as KnownArchives, { release: 'FireSquid' }),
+  })
+  .addCallHandler('Staking.nominate', stakingNominateCall(network, 0))
+  .addEventHandler(
+    'Staking.Bonded',
+    stakingActionEvent(network, 0, SubstrateStakingActionType.Bonded)
```

Comment:
The `0` is the token index, I got rid in balances and just hard coded it where it was used in the registry code, it's always 0 in reality. Just an opinion though, maybe it's better to leave it in case we ever need it